boat_interfaces/src/tacking_action.py

```diff
@@ -83,7 +56,10 @@ def tacking_callback(self, goal):
 		self._feedback.status = "Entered Action Callback"
 		
 		# publish info to the console for the user
-		rospy.loginfo('Tacking Action: Tacking Action Startup')
+		if goal.direction == 1:
```

Comment:
Had an idea, and maybe it doesn't need to through for this MR, but I think the utility should have like an enum section where we can add like `direction.clockwise = 1` `direction.counterclockwise = -1` and then this line could become `goal.direction == direction.clockwise`

Reply:
I agree that we should define the value better, though I was planning on implementing it as a constant in the message file. Planning on doing this in separate PR. I've opened #22 to track this.